tests/labhub_test.py

```diff
@@ -47,62 +53,67 @@ def test_invite_cmd(self):
             'coala developers': mock_team_developers
         }
 
-        labhub, testbot = plugin_testbot(plugins.labhub.LabHub, logging.ERROR)
-        labhub.activate()
-        labhub._teams = teams
+        mock_dict = {
+            'TEAMS': teams,
+            'is_room_member': MagicMock(),
+        }
+        labhub = self.load_plugin(
```

Comment:
load plugin only once, not in each test case. There is no `tearDown` unloading the plugin. Load the plugin in `setUp` with global mocks that are required everywhere. and then directly use `self.inject_mocks` for subsequent mock injection.

Applies for everywhere.